_tests/browser_tests/jenkins_conf.js

```diff
@@ -0,0 +1,41 @@
+var env = require('./environment.js');
+
+exports.config = {
+  framework: 'jasmine2',
+  specs: ['spec_suites/shared/*.js'],
+  capabilities: {
+    'browserName': 'chrome',
+    'name': 'flapjack-browser-tests'
+  },
+  
+  sauceUser: process.env.SAUCE_USER,
+  sauceKey: process.env.SAUCE_KEY,
+
+  onPrepare: function() {
+    return browser.ignoreSynchronization = true;
+
+
+    var jasmineReporters = require('jasmine-reporters');
+    var mkdirp = require('mkdirp');
+    var SpecReporter = require('jasmine-spec-reporter');
+
+    // add jasmine spec reporter
+    jasmine.getEnv().addReporter(new SpecReporter({displayStacktrace: true}));
```

Comment:
Ideally run this file through the eslint linter as there will be some minor stylistic violations here around spaces inside the parens. (Refer to https://github.com/cfpb/cfgov-refresh/pull/745)